### PR TITLE
Fix wrong indent and a typo in Makefile

### PR DESCRIPTION
**Pull request by TaKO8Ki**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ ifdef BUILD_PLATFORM
 endif
 
 ifdef PHTHON_PATH
```
Comment (nghialv) [inline Makefile:18]: Oh, there is a typo here, can you fix it in this pull request?
"PHTHON" -> "PYTHON"

Comment (TaKO8Ki) [reply Makefile]: I've just fixed it.

**Comment by nghialv** [comment]
Thank you.
/approve

**Pull request merged**